src/liboslnoise/oslnoise_test.cpp

```diff
@@ -19,372 +19,389 @@ using namespace OSL::oslnoise;
-        OIIO_CHECK_EQUAL_THRESH (vs4, vresults_4d[i], eps);
+        OIIO_CHECK_EQUAL_THRESH(vs1, vresults_1d[i], eps);
+        OIIO_CHECK_EQUAL_THRESH(vs2, vresults_2d[i], eps);
+        OIIO_CHECK_EQUAL_THRESH(vs3, vresults_3d[i], eps);
+        OIIO_CHECK_EQUAL_THRESH(vs4, vresults_4d[i], eps);
     }
 
     // Make sure the right thing happens at integer cell boundaries.
     // hashnoise should be discontinuous everywhere!
-    OIIO_CHECK_NE (hashnoise( 0.9999f), hashnoise( 1.0f));
-    OIIO_CHECK_NE (hashnoise( 1.0001f), hashnoise( 1.0f));
-    OIIO_CHECK_NE (hashnoise(-0.0001f), hashnoise( 0.0f));
-    OIIO_CHECK_NE (hashnoise( 0.0001f), hashnoise( 0.0f));
-    OIIO_CHECK_NE (hashnoise(-1.0001f), hashnoise(-1.0f));
-    OIIO_CHECK_NE (hashnoise(-0.9999f), hashnoise(-1.0f));
+    OIIO_CHECK_NE(hashnoise(0.9999f), hashnoise(1.0f));
+    OIIO_CHECK_NE(hashnoise(1.0001f), hashnoise(1.0f));
+    OIIO_CHECK_NE(hashnoise(-0.0001f), hashnoise(0.0f));
+    OIIO_CHECK_NE(hashnoise(0.0001f), hashnoise(0.0f));
+    OIIO_CHECK_NE(hashnoise(-1.0001f), hashnoise(-1.0f));
+    OIIO_CHECK_NE(hashnoise(-0.9999f), hashnoise(-1.0f));
 
     if (make_images) {
-        MAKE_IMAGE (hashnoise);
+        MAKE_IMAGE(hashnoise);
     }
 
     // Time trials
     Benchmarker bench;
-    float fval = 0.5f; clobber (fval);
-    Vec3 vval (0,0,0); clobber (vval);
-    bench ("  hashnoise(f)", [&](){ DoNotOptimize (hashnoise<float>(fval)); });
-    bench ("  hashnoise(f,f)", [&](){ DoNotOptimize (hashnoise<float,float>(fval, fval)); });
-    bench ("  hashnoise(v)", [&](){ DoNotOptimize (hashnoise<const Vec3&>(vval)); });
-    bench ("  hashnoise(v,f)", [&](){ DoNotOptimize (hashnoise<const Vec3&,float>(vval, fval)); });
-
-    bench ("  vhashnoise(f)", [&](){ DoNotOptimize (vhashnoise<float>(fval)); });
-    bench ("  vhashnoise(f,f)", [&](){ DoNotOptimize (vhashnoise<float,float>(fval, fval)); });
-    bench ("  vhashnoise(v)", [&](){ DoNotOptimize (vhashnoise<const Vec3&>(vval)); });
-    bench ("  vhashnoise(v,f)", [&](){ DoNotOptimize (vhashnoise<const Vec3&,float>(vval, fval)); });
+    float fval = 0.5f;
+    clobber(fval);
+    Vec3 vval(0, 0, 0);
+    clobber(vval);
+    bench("  hashnoise(f)", [&]() { DoNotOptimize(hashnoise<float>(fval)); });
+    bench("  hashnoise(f,f)",
+          [&]() { DoNotOptimize(hashnoise<float, float>(fval, fval)); });
+    bench("  hashnoise(v)",
+          [&]() { DoNotOptimize(hashnoise<const Vec3&>(vval)); });
+    bench("  hashnoise(v,f)",
+          [&]() { DoNotOptimize(hashnoise<const Vec3&, float>(vval, fval)); });
+
+    bench("  vhashnoise(f)", [&]() { DoNotOptimize(vhashnoise<float>(fval)); });
+    bench("  vhashnoise(f,f)",
+          [&]() { DoNotOptimize(vhashnoise<float, float>(fval, fval)); });
+    bench("  vhashnoise(v)",
+          [&]() { DoNotOptimize(vhashnoise<const Vec3&>(vval)); });
+    bench("  vhashnoise(v,f)",
+          [&]() { DoNotOptimize(vhashnoise<const Vec3&, float>(vval, fval)); });
 }
```

Comment:
This was more readable prior to clang-format.

Reply:
fixing